cli/README-cn.md

```diff
@@ -0,0 +1,719 @@
+# CodePush 命令行
+
+CodePush是一个云服务，它能让Cordova和React Native的开发者将手机应用的更新直接部署到用户的设备上。
```

Comment:
Perhaps we should add a note at the top that says that this doc is community contributed and maintained?